### DISCUSSION TITLE
[M-Z] (2024/25) - sez. 0.3 - es. 5 (7 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#276] [Esercizi M-Z]
![es.5](https://github.com/user-attachments/assets/43e69c00-f1d3-47a4-826e-be723c476320)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/b3be8456-3334-46be-ab7a-9dffd0415634)
